SoftLayer/CLI/sshkey/add.py

```diff
@@ -40,4 +40,12 @@ def cli(env, label, in_file, key, note):
     mgr = SoftLayer.SshKeyManager(env.client)
     result = mgr.add_key(key_text, label, note)
 
-    env.fout("SSH key added: %s" % result.get('fingerprint'))
+    table = formatting.KeyValueTable(['name', 'value'])
+    table.align['name'] = 'r'
+    table.align['value'] = 'l'
+
+    table.add_row(['id', result['id']])
```

Comment:
Is it possible that some of the key don't exist? Will it output empty string?

Reply:
This happens when you add a new key. So this scenario should never occur..